src/Workleap.Extensions.Mongo/Indexing/MongoIndexer.cs

```diff
@@ -94,6 +96,11 @@ private async Task UpdateIndexesInternalAsync(IEnumerable<Type> types, IMongoDat
     {
         var registry = new IndexRegistry(types);
 
+        foreach (var cachedIndexType in MongoReflectionCache.GetIndexProviderTypes())
```

Comment:
Hmmm wondering if there is potential pitfall to automatically append those cached indexes every times. Should it be opt-in when you pass Assembly 🤔. What do you think? Could also be worth to get someone else feedback

Reply:
If you dont use Configuration its not going to append anything. But if you, for some reason,  called UpdateIndexes multiple times, we would want the Configuration based indexes all the time.